1-js/01-getting-started/2-code-editors/article.md

```diff
@@ -51,12 +51,12 @@ I'm using:
 - [WebStorm](http://www.jetbrains.com/webstorm/) for JS, and if there is one more language in the project, then I switch to other Jetbrains editors like [PHPStorm](http://www.jetbrains.com/phpstorm/) (PHP), [IDEA](http://www.jetbrains.com/idea/) (Java), [RubyMine](http://www.jetbrains.com/ruby/) (Ruby). There are editors for other languages too, but I haven't used them.
 - As a lightweight editor -- [Sublime Text](http://www.sublimetext.com) or [Atom](https://atom.io/).
 
-If you don't know what to choose -- you can consider these ones.
+If you don't know what to choose, you can consider these ones.
 
 ## Let's not argue
 
-The editors in the lists above are those that me or my friends -- good developers have been using for a long time and are happy with.
+The editors in the lists above are those that either my friends or I -- good developers -- have been using for a long time and are happy with.
```

Comment:
I see what you mean about wanting to clarify that 'good developers' is meant to refer only to your friends, rather than your friends and you.

But putting "-- good developers --" aside for a moment, there is a grammatical problem with "me or my friends". 

Me" is an **object pronoun**, but in the sentence above, "me or my friends" is the **subject** of the verb phrase "have been using". So first we need to change the **object pronoun** "me" to a **subject pronoun:** "I or my friends". Furthermore, it is a convention of English usage to place an instance of "I" at the end of a sequence of subjects. So the final refactoring would be "my friends or I".

"My friends or I" is grammatically correct and consistent with common English usage, but in terms of style it is definitely up to you where "-- good developers --" belongs. It could read: "... that either my friends -- good developers -- or I have..."

Let me know how you'd like me to proceed and I will correct the sentence.

Cheers!